dracut/30ignition/ignition-generator

```diff
@@ -95,4 +98,5 @@ if [[ $(systemd-detect-virt || true) =~ ^(kvm|qemu)$ ]]; then
     oem_id=qemu
 fi
 
-echo "OEM_ID=$(cmdline_arg coreos.oem.id ${oem_id})" > /run/ignition.env
+echo "OEM_ID=$platform_id" > /run/ignition.env
```

Comment:
Leaving for backwards compat reasons?

Reply:
Yeah, though looking more closely now, we're the only users of `/run/ignition.env`, so I think we can drop this.

Reply:
OK, I pushed another version which drops this, though I'm totally open to keeping it around for a while and just dropping it at the same time we eventually drop `coreos.platform.id`.